pypesto/select/model_problem.py

```diff
@@ -122,12 +122,13 @@ def __init__(
                     )
                 # TODO rename `minimize_options` to `minimize_kwargs`.
                 # TODO or allow users to provide custom `minimize` methods?
-                elif minimize_options:
+                else:
```

Comment:
What happens now if minimise_options is None?

Reply:
A default value of `{}` was already being set.

https://github.com/ICB-DCM/pyPESTO/blob/45e5ac09fe8c1f45cccfdd392c9133a58a9ef8c0/pypesto/select/model_problem.py#L86-L88